app/templates/community/index.hbs

```diff
@@ -1,43 +1,52 @@
-<div class="container">
+<section class="container layout" aria-labelledby="section-community">
   <h1>Get Involved: Join the Growing Ember.js Community</h1>
-  <section class="margin-top-large" aria-labelledby="section-community">
-    <div class="layout-grid">
-      <div class="col-2-large">
-        <img alt="" role="presentation" class="img-fluid" src="/images/community/sos.png">
-      </div>
-      <div class="col-4-large">
-        <h2 id="section-community" class="margin-bottom-small">Stuck? Lost? Get Help from the Community</h2>
-        <p>
-          <a href="http://discuss.emberjs.com">Ember Discussion Forum</a> &mdash; a great venue for discussing features, architecture and best practices.
-        </p>
-        <p>
-          <a href="https://discordapp.com/invite/zT3asNS">Ember Community Discord</a> &mdash; ask questions and chat with community members in real-time. (Looking for the Ember Community Slack? We recently switched away from Slack to <a href="https://discordapp.com">Discord</a>. Read more about why <a href="https://github.com/emberjs/rfcs/pull/345">here</a>).
-        </p>
-        <p>
-          <a href="http://stackoverflow.com/questions/tagged/ember.js">StackOverflow</a> &mdash; used to track questions. Just
-          tag your question with <code>ember.js</code> or <a href="http://stackoverflow.com/questions/tagged/ember.js">search for questions with that tag</a>. Please check to see if your question has already been answered before asking a new one.
-        </p>
-        <p>
-          Please note that these resources are not managed or moderated by the Ember Core Team. They are public forums.
-        </p>
-      </div>
-    </div>
-  </section>
-  <section class="margin-top-medium" aria-labelledby="section-latest-news">
-    <h2 id="section-latest-news">Stay Up to Date with the Latest News</h2>
+
+  <div class="lg:col-2">
+    <img alt="" role="presentation" class="img-fluid" src="/images/community/sos.png">
+  </div>
+
+  <div class="lg:col-4">
+    <h2 id="section-community">Stuck? Lost? Get Help from the Community</h2>
```

Comment:
the ID should be removed here- since the page H1 is in this section, that's the one that takes precedence.